lib/benchee/configuration.ex

```diff
@@ -334,7 +334,7 @@ defmodule Benchee.Configuration do
   defp update_measure_memory(config = %{memory_time: memory_time}) do
     otp_version = List.to_integer(:erlang.system_info(:otp_release))
 
-    if (memory_time > 0) and otp_version <= 18 do
+    if memory_time > 0 and otp_version <= 18 do
```

Comment:
really? I dunno I like to put parentheses explicity so I don't have to worry about whether and binds stronger `>` or not

Reply:
I can see leaving parens in boolean expressions in as a good feature request for the formatter. Maybe you can ping them on the mailing list?